### DISCUSSION TITLE
UnknowOS Message in Longhorn Node Condition on RHEL

### DISCUSSION THREAD

**Discussion by jangseon-ryu** [longhorn/longhorn#9823] [Q&A] [answered]
Hello Longhorn community,

I encountered an issue during testing for the adoption of Longhorn. The Longhorn Node Condition displayed a message stating UnknowOS.
```
  - lastProbeTime: ""
    lastTransitionTime: "2024-08-29T06:35:03Z"
    message: Unable to verify the required packages because the OS image 'red hat
      enterprise linux 8.8 (ootpa)' is unknown to the Longhorn system. Please ensure
      the required packages are installed.
    reason: UnknownOS
    status: "False"
    type: RequiredPackages
```

I’m wondering if this means RHEL is not officially supported by Longhorn. If it is supported, would it be acceptable for me to attempt modifying the code and contributing the changes back to the project?

Any guidance or clarification would be greatly appreciated.

Best regards,
Jason

**Comment by derekbit** [comment, marked as answer]
Longhorn supports RHEL. It is a bug. No worries for the warning message because it won't impact any the functionalities of Longhorn.

If you are interested in the contribution, you can submit a PR for fixing it. Thank you.

cc @mantissahz @longhorn/qa please remember to verify the OSes listed in https://longhorn.io/docs/1.7.2/best-practices/. Thanks.
